frontend/routes/notifications/notifications.panel.tsx

```diff
@@ -0,0 +1,70 @@
+
+/**
+ *  Copyright (C) 2018 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import * as React from "react";
```

Comment:
single-quotes

Reply:
Shouldnt there be a lint-rule for enforcing that?

Reply:
Yes, definitely!

Reply:
@simplypixi Can you guys set up a linter rule for this? (Also anything like indentation, spaces around bracketing etc) would be very hard to fully enforce/spot going forward otherwise!

Reply:
@carmenfan Yes, sure. I'll do that after migration of login page. ;)